prov/efa/test/efa_unit_tests.c

```diff
@@ -0,0 +1,14 @@
+#include "efa_unit_tests.h"
+
+int main(void)
+{
+	int ret;
+	const struct CMUnitTest efa_unit_tests[] = {
+		cmocka_unit_test(create_efa_ah_twice)    // Requires an EFA device to work
```

Comment:
Let's stick with the C-style comments :-)

Reply:
done :)